keyboards/keebio/iris/keymaps/boo/keymap.c

```diff
@@ -0,0 +1,245 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _DVORAK,
+    _QWERTY,
+    _CODE,
+};
+
+// work in progress simulation of kakoune behavior
+bool kak_mode = false;
+
+enum tapdances {
+    _LCTLGUI,
+};
+
+enum keycodes {
+    KC_KAK = SAFE_RANGE,
+};
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [_LCTLGUI] = ACTION_TAP_DANCE_DOUBLE(KC_LCTL, KC_LGUI),
+};
+
+// Layers
+#define KC_DVORK DF(_DVORAK)
+#define KC_QWRTY DF(_QWERTY)
+#define KC_CODE MO(_CODE)
+
+// Modifiers
+#define KC_COPY LCTL(KC_C)
+#define KC_PASTE LCTL(KC_V)
+
+// Tapdances
+#define KC_LCG TD(_LCTLGUI)
+
+// Misc
+#define KC_ KC_TRNS
+#define KC_RESET RESET
+
+// Used to create a keymap using only KC_ prefixed keys
+#ifndef LAYOUT_kc
+#define LAYOUT_kc( \
+    L00, L01, L02, L03, L04, L05,           R00, R01, R02, R03, R04, R05, \
+    L10, L11, L12, L13, L14, L15,           R10, R11, R12, R13, R14, R15, \
+    L20, L21, L22, L23, L24, L25,           R20, R21, R22, R23, R24, R25, \
+    L30, L31, L32, L33, L34, L35, LT4, RT4, R30, R31, R32, R33, R34, R35, \
+                        LT1, LT2, LT3, RT3, RT2, RT1 \
+    ) \
+    LAYOUT( \
+        KC_##L00, KC_##L01, KC_##L02, KC_##L03, KC_##L04, KC_##L05,                     KC_##R00, KC_##R01, KC_##R02, KC_##R03, KC_##R04, KC_##R05, \
+        KC_##L10, KC_##L11, KC_##L12, KC_##L13, KC_##L14, KC_##L15,                     KC_##R10, KC_##R11, KC_##R12, KC_##R13, KC_##R14, KC_##R15, \
+        KC_##L20, KC_##L21, KC_##L22, KC_##L23, KC_##L24, KC_##L25,                     KC_##R20, KC_##R21, KC_##R22, KC_##R23, KC_##R24, KC_##R25, \
+        KC_##L30, KC_##L31, KC_##L32, KC_##L33, KC_##L34, KC_##L35, KC_##LT4, KC_##RT4, KC_##R30, KC_##R31, KC_##R32, KC_##R33, KC_##R34, KC_##R35, \
+                                                KC_##LT1, KC_##LT2, KC_##LT3, KC_##RT3, KC_##RT2, KC_##RT1 \
+    )
+#endif
```

Comment:
```suggestion
```

Layouts which prefix keys with KC_ are discouraged now, as they do not play well with the Configurator.

Reply:
Fixed